frontend/src/components/individual-recipe/Tutorial.js

```diff
@@ -27,18 +27,34 @@ class Tutorial extends Component {
     super(properties);
     this.state = {
       isLastStep: false,
-      isSpeakerOff: false,
     };
+
     this.switchSpeaker = this.switchSpeaker.bind(this);
+    this.getSelectedStep = this.getSelectedStep.bind(this);
   }
 
   componentDidMount() {
     this.noteIfLastStep();
+    if (
+      !this.state.showModal &&
+      this.props.isSpeakerOn &&
+      this.props.isTutorialActive
+    ) {
```

Comment:
I personally think this looks weird. Left a suggestion but I didn't find specific guidelines so feel free to choose what you prefer.

```suggestion
    if (!this.state.showModal &&
        this.props.isSpeakerOn &&
        this.props.isTutorialActive) {
```